packages/destination-actions/src/destinations/salesforce/lead/index.ts

```diff
@@ -0,0 +1,98 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const API_VERSION = 'v53.0'
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Lead',
+  description: 'Lead action',
+  fields: {
+    crud: {
+      label: 'CRUD Operation',
+      description: 'CRUD',
+      type: 'string',
+      required: true,
+      choices: [
+        { label: 'Create', value: 'create' },
+        { label: 'Update', value: 'update' },
+        { label: 'Upsert', value: 'upsert' }
```

Comment:
```suggestion
        { label: 'Create', value: 'create' }
```

I think better to just not support these yet rather than have placeholders. Would rather have thin slices of fully implemented so it's clear as we start testing it what is expected to work.